img2dataset/downloader.py

```diff
@@ -38,9 +38,10 @@ def download_image(row, timeout, user_agent_token, disallowed_header_directives)
     """Download an image with urllib"""
     key, url = row
     img_stream = None
-    user_agent_string = "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0"
+    user_agent_string = "img2dataset/1.x ("
```

Comment:
Any reason not into use `{user_agent_token}` rather than hard coding img2dataset here?

Reply:
The reference to the repository was hardcoded previously if any user-agent was specified, so it seemed appropriate to use it as the base tool name with the user-provided string added in the comment section.

default UA: `img2dataset/1.x (compatible; img2dataset; +https://github.com/rom1504/img2dataset)`
user-provided UA: `img2dataset/1.x (compatible; <user-provided>; +https://github.com/rom1504/img2dataset)`
empty UA: `img2dataset/1.x (+https://github.com/rom1504/img2dataset)`

previous strings were:
default UA: `Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0 (compatible; img2dataset; +https://github.com/rom1504/img2dataset)`
user-provided UA: `Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0 (compatible; <user-provided>; +https://github.com/rom1504/img2dataset)`
empty UA: `Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0`